build-logic/detekt/src/main/resources/detekt.yml

```diff
@@ -6,6 +6,13 @@ style:
   ThrowsCount:
     active: false
 
+formatting:
+  active: true
+  TrailingComma:
+    active: true
+    allowTrailingComma: true
+    allowTrailingCommaOnCallSite: false
```

Comment:
should we enforce this as well since there's no transpiling issue in kotlin?

Reply:
it's optional in kotlin, so it's good to have a rule.

Reply:
prefer always with commas

Reply:
updated